src/components/Editor/Tabs.css

```diff
@@ -1,62 +1,56 @@
 .source-header {
   border-bottom: 1px solid var(--theme-splitter-color);
+  width: 100%;
   height: 30px;
   display: flex;
-  flex: 1;
-  flex-flow: row wrap;
-  -webkit-align-items: stretch;
-  align-items: stretch;
+  justify-content: space-between;
 }
 
 .source-header * {
   -moz-user-select: none;
   user-select: none;
 }
 
-.source-tabs {
-  max-width: calc(100% - 80px);
-  float: left;
-  margin-inline-start: 21px;
-}
-
 .source-header .new-tab-btn {
-  width: 14px;
-  height: 14px;
-  display: inline-block;
-  position: relative;
-  top: 4px;
-  margin: 4px;
-  margin-inline-start: 8px;
-  line-height: 0;
+  align-self: flex-end;
+  padding: 4px;
   cursor: pointer;
   fill: var(--theme-comment);
   transition: 0.1s ease;
 }
 
+.source-header .new-tab-btn svg {
+  width: 12px;
+}
+
+.source-tabs {
+  max-width: calc(100% - 80px);
+  align-self: flex-end;
+  display: flex;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+}
+
 .source-tab {
   color: var(--theme-faded-tab-color);
   border: 1px solid transparent;
+  border-bottom: 0; 
   border-top-left-radius: 2px;
   border-top-right-radius: 2px;
-  height: 24px;
-  line-height: 20px;
-  display: inline-block;
   position: relative;
   transition: all 0.25s ease;
-  min-width: 40px;
-  overflow: hidden;
-}
-
-.source-tab {
```

Comment:
These rules are merged. But looks like a bug from `postcss-bidirection`. If I explicitly don't specify these rules https://github.com/devtools-html/debugger.html/pull/1976/files#diff-d9abd0398fe2e0fa5793ba7d100231f5R52, the `hover` is never applied, because the rules are suppressed by the postcss processing. Maybe that's why, these CSS rules weren't merged and written out explicitly.